### DISCUSSION TITLE
Add PackedVector2iArray and PackedVector3iArray

### DISCUSSION THREAD

**Discussion by acgc99** [godotengine/godot-proposals#5713] [Engine Core]
Since `Vector2i` and `Vector3i` were added, I lack the equivalent packed arrays: `PackedVector2iArray` and `PackedVector3iArray `.

I'm using Godot v4.0.beta3.official [01ae26d31] and if I try to set  a value of a `PackedVector2iArray` to a `Vector2i` I get an error (expected):
`Invalid set index '0' (on base: 'PackedVector2Array') with value of type 'Vector2i'`

I would really appreciate the addition of these arrays, since I'm working with `Vector2i` for tiles positions to avoid floating point errors.

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/5486.
